addon/components/pie-chart/component.js

```diff
@@ -15,10 +15,15 @@ export default BaseChartComponent.extend({
 
     externalLabels: false,
     labels: true,
+    formatter: d => d,
 
     buildChart() {
-        let chart = dc.pieChart(`#${this.get('elementId')}`);
+        if (this.get('group').all().length == 0) {
```

Comment:
Should use `getWithDefault` or add a falsey check on `group`. And use `===` instead of `==`